app/assets/stylesheets/webfonts/knockout-font.scss

```diff
@@ -0,0 +1,11 @@
+@font-face{
+  font-family: "Knockout-48Featherweight";
+  src: asset-url("/webfonts/fonts/Knockout-48.otf") format('otf');
+  font-weight: normal;
+  font-style: normal;
+}
+
+.knockout-font{
+  font-family: "Knockout-48Featherweight", "Open Sans", Helvetica, Arial, sans-serif;
+  font-weight: 400;
```

Comment:
I think we will probably want to add `text-transform: uppercase;` since we never use lower case knockout.

Reply:
Yeah, I debated whether or not to add it, so this confirms I should have. Thanks!